.travis.yml

```diff
@@ -43,7 +43,7 @@ env:
         - CONDA_DEPENDENCIES='requests beautifulsoup4 html5lib keyring aplpy matplotlib pyregion jinja2 flask'
         # pandas is used as a workaround of an astropy issue, sometimes it's required for the remote tests
         - CONDA_DEPENDENCIES_REMOTE='requests beautifulsoup4 html5lib keyring aplpy matplotlib pyregion jinja2 flask pandas'
-        - PIP_DEPENDENCIES='https://github.com/keflavich/httpbin/archive/master.zip pytest-dependency'
+        - PIP_DEPENDENCIES='https://github.com/keflavich/httpbin/archive/master.zip regions astropy_healpix mocpy pytest-dependency'
```

Comment:
`regions` and `astropy-healpix` are conda packaged and are available on the astropy channel, so would prefer if they were to be added as conda dependencies above.

Reply:
I stay corrected, healpix is not yet available on the astropy channel, but I'm about to fix it.